go-server-server/go/default.go

```diff
@@ -1477,20 +1532,32 @@ func ConfigVrfVrfIdRoutesGet(w http.ResponseWriter, r *http.Request) {
         return
     }
 
-    db := &conf_db_ops
+    // Check non-persistent static routes first
+    db := &app_db_ops
     var pattern string
+    var persistent bool
 
-    rt_tb_name := STATIC_ROUTE_TB
-
-    pattern = generateDBTableKey(db.separator, rt_tb_name, vrf_id_str, ipprefix)
-    routes := []RouteModel{}
-
+    persistent = false
+    pattern = generateDBTableKey(db.separator, STATIC_ROUTE_TB, vrf_id_str, ipprefix)
     kv, err := GetKVsMulti(db.db_num, pattern)
     if err != nil {
         WriteRequestError(w, http.StatusInternalServerError, "Internal service error", []string{}, "")
         return
     }
 
+    if len(kv) == 0 {
```

Comment:
I think for "GET", we should fetch both app_db and config_db. Client can identify it based on "persistent" field.

Reply:
We are checking both DBs here. The GET API does not take any parameters. Since mostly our use-case is non-persistent, I check the APP DB first and then if I can't find anything, I check the CONFIG DB. Since the VRF name is going to be unique, we should be good here.